posts/2021-05-11-database-ssl-primer.adoc

```diff
@@ -335,12 +348,11 @@ javax.net.ssl|DEBUG|41|Default Executor-thread-9|CertificateMessage.java:321|Pro
 )
 ----
 
-Driver sent certificate, but database could not verify:
+In other cases, the driver sends a certificate, but database cannot not verify it. This issue is difficult to debug because the issue is on the database side. The driver might throw one of many different exceptions to indicate that it could not finish the handshake.
 
-This is a difficult issue to debug because the issue is on the database side and the driver may throw one of many different exception to indicate that it could not finish the handshake.  
-The following is a sample exception of when this occures when trying to connect to a SQLServer database. 
+The following sample exception shows an error message when the driver tries to connect to an SQLServer database that the database cannot verify:
 
 [source,txt]
 ----
 java.sql.SQLException: The driver could not establish a secure connection to SQL Server by using Secure Sockets Layer (SSL) encryption. Error: "SQL Server did not return a response. The connection has been closed. DSRA0010E: SQL State = 08S01, Error Code = 0
-----
\ No newline at end of file
+----
```

Comment:
can we suggest a user action? Or where else to look to debug errors like this?

Reply:
I would say the resolution would be to:
"Check with your Database Administrator to see if logs were produced by the database that can indicate the reason the database did not verify the certificate.  Similar to the client side this could be because a truststore location was mis-configured, or did not contain the correct certificate."

Reply:
updated